app/helpers/otp_helper.rb

```diff
@@ -0,0 +1,44 @@
+module OtpHelper
+  def phone_confirmation_instructions
+    if @delivery_method == 'sms'
+      t('instructions.2fa.confirm_code_sms', number: @phone_number)
+    elsif @delivery_method == 'voice'
+      t('instructions.2fa.confirm_code_voice', number: @phone_number)
+    end
+  end
+
+  def alternative_2fa_links
+    case @delivery_method
+    when 'sms'
+      "#{voice_fallback_link}#{totp_option_link}."
```

Comment:
I chose to do interpolation because the second link is optional, and I'm not sure of a better way to compose i18n strings. 

In your proposed example above, wouldn't we still need to use the `==`  helper to display each `link`? Even if we iterate over an array of strings, there are still `<a>` tags embedded in them that need to be interpolated.